apps/studio/src/pages/home/library/MarketplaceTab/StartSaleModal.tsx

```diff
@@ -0,0 +1,195 @@
+import { FunctionComponent } from "react";
+import { Box, Stack } from "@mui/material";
+import { Button, HorizontalLine, Modal, Typography } from "@newm-web/elements";
```

Comment:
Typography from NEWM elements instead of MUI pattern, should I create a ticket to just transition us to MUI for this one element across the code base?